lib/mongoid/history/attributes/update.rb

```diff
@@ -13,6 +13,7 @@ def attributes
               @attributes[k] = format_field(k, v)
             end
           end
+          insert_embeds_one_changes_on_child if trackable_class.tracked_embeds_one.present? && changes.empty?
```

Comment:
Seems strange that you'd have the `changes.empty?` check here. Maybe write a spec that changes fields, then embeds, then both. Then we're keeping things in `@attributes`, so I imagine this really should be something like 

```ruby
@attributes ||= begin
   changes_in_this_instance_or_something_like_that  + changes_in_embeds + ...
end
```